buildkite/bazelci.py

```diff
@@ -633,6 +633,10 @@
 
 AUTO_DIFFBASE_VALUES = frozenset(["1", "true", "auto"])
 
+# Always run all test targets if any of the paths here are modified by the current commit.
+# Values can be directory paths (with a trailing slash) or file paths.
+DISABLE_BAZEL_DIFF_IF_MODIFIED = (".bazelci/", ".bazelversion")
```

Comment:
Should we add `.bazelrc` file as well?

Reply:
I assume most changes should have an impact on the results returned by query, right?